src/services/goToDefinition.ts

```diff
@@ -32,6 +32,13 @@ namespace ts.GoToDefinition {
             return functionDeclaration ? [createDefinitionFromSignatureDeclaration(typeChecker, functionDeclaration)] : undefined;
         }
 
+        if (node.kind === SyntaxKind.CaseKeyword || node.kind === SyntaxKind.DefaultKeyword) {
+            const switchStatement = findAncestor(node.parent, isSwitchStatement) as SwitchStatement | undefined;

```

Comment:
You shouldn't need the type assertion anymore.

```suggestion
            const switchStatement = findAncestor(node.parent, isSwitchStatement);
```